package.json

```diff
@@ -40,6 +40,22 @@
       "title": "Custom IPFS API Port",
       "type": "integer",
       "value": 5001
+    },
+    {
+      "name": "rewriteAs",
+      "type": "menulist",
+      "title": "Rewrite IPFS URIs in the UI as",
+      "value": "gateway",
+      "options": [
+        {
+          "value": "gateway",
```

Comment:
I feel a better value here would be "publicGateway" so that there is no overlap between preference name and its value.